x-pack/plugins/spaces/public/management/edit_space/edit_space_general_tab.tsx

```diff
@@ -137,7 +143,7 @@ export const EditSpaceSettingsTab: React.FC<Props> = ({ space, features, history
       setIsLoading(true);
 
       let disabledFeatures: string[] | undefined;
-      if (spaceClone.solution === SOLUTION_VIEW_CLASSIC) {
+      if (spaceClone.solution == null || spaceClone.solution === SOLUTION_VIEW_CLASSIC) {
```

Comment:
If the space was created via an API, then the solution field could be blank. The SpaceValidator treats that as a non-issue, and this code change follows that standard.

Reply:
It is also the case for all spaces created in Kibana before the `8.16` release.